vault/identity_store_oidc.go

```diff
@@ -138,21 +138,44 @@ func oidcPaths(i *IdentityStore) []*framework.Path {
 	return []*framework.Path{
 		{
 			Pattern: "oidc/config/?$",
+
+			DisplayAttrs: &framework.DisplayAttributes{
+				OperationPrefix: "oidc",
+			},
+
 			Fields: map[string]*framework.FieldSchema{
 				"issuer": {
 					Type:        framework.TypeString,
 					Description: "Issuer URL to be used in the iss claim of the token. If not set, Vault's app_addr will be used.",
 				},
 			},
-			Callbacks: map[logical.Operation]framework.OperationFunc{
-				logical.ReadOperation:   i.pathOIDCReadConfig,
-				logical.UpdateOperation: i.pathOIDCUpdateConfig,
+
+			Operations: map[logical.Operation]framework.OperationHandler{
+				logical.ReadOperation: &framework.PathOperation{
+					Callback: i.pathOIDCReadConfig,
+					DisplayAttrs: &framework.DisplayAttributes{
+						OperationSuffix: "configuration",
+					},
+				},
+				logical.UpdateOperation: &framework.PathOperation{
+					Callback: i.pathOIDCUpdateConfig,
+					DisplayAttrs: &framework.DisplayAttributes{
+						OperationVerb: "configure",
```

Comment:
Does this also need the `configuration` suffix

Reply:
I left the suffix intentionally blank here to make it "oidc-configure" since it's configuring OIDC.